pkg/image/resolve.go

```diff
@@ -96,39 +92,87 @@ func GetImages(systemChartPath, chartPath string, externalImages map[string][]st
 	return imagesList, imagesAndSourcesList, nil
 }
 
-func setImages(source string, imagesFromArgs []string, imagesSet map[string]map[string]bool) {
+func AddImagesToImageListConfigMap(cm *v1.ConfigMap, rancherVersion, systemChartsPath string) (err error) {
```

Comment:
I will make this change in the next commit. This function had minor modifications, but it shows up as if I added it because I re-organized resolve.go to place the most important functions at the top.